integrations/unity-csharp/README.md

```diff
@@ -0,0 +1,130 @@
+# VDK Unity
+
+This project demonstrates the use of Euclideon Vault developer Kit (VDK) with the Unity game engine. 
+
+<!-- TODO: Fill this section below with metadata about this sample-->
+```
+Language:              C#
+Type:                  Integration
+Contributor:           Euclideon Vault Development Team <support@euclideon.com>
+Organization:          Euclideon, https://euclideon.com/vault
+Date:                  2020-02-03
+Vault SDK Version:     0.5.0
+Toolsets:              Requires Unity >=2019.3
+```
+
+## Resources Required
+Unity Postprocessing >=v2.3.0
+
+## How to use the sample
+<!-- TODO: Explain how this sample can be used and what is required to get it running -->
+There are two projects included:
+
+* The basic package, including a basic flight camera and collider, this demonstrates the basic concepts required to get simulations and games using VDK running in unity. This is ideal for developers wishing to use VDK with their own projects
+* The advanced project: includes a driving simulator and third person ragdoll. This demonstrates use of VDK with Unity physics and is good for exploring and testing interactions of unity systems with Unlimited Detail point clouds.
+
+**Both examples require a valid license for Euclideon Vault SDK, trial licenses can be obtained [here](https://zfrmz.com/gwVUru84d60yUedxmLx9/?ref=Unity%20Sample%20Code)** 
+
+### Installation
+
+Both examples are tested with Unity 2019.3.0f6, all steps must be repeated for each package in their respective directories.
+
+1. Download and extract VDK 0.5.0 package from [here](https://earth.vault.euclideon.com) using your license credentials (if you do not have one, free trials are available from [here](https://zfrmz.com/gwVUru84d60yUedxmLx9/?ref=Unity%20Sample%20Code) )
+2. Download the Unity VDK example from [here]()
+3. Copy the files from Euclideon_vdk0.5.0/lib/(_your operating system here_)/to Assets/VDK in your Unity project
+4. Replace the username and password lines from _Assets/VDK/vdkLogin.cs_ to your vault login details **YOUR UNITY EDITOR MAY CRASH ON LOAD IF THIS STEP IS NOT COMPLETED**
```

Comment:
We probably shouldn't merge this with a known crash.

Reply:
This doesn't appear to occur any more anyway